README.md

```diff
@@ -74,6 +74,42 @@ Edools::Enrollment.update(id: 1, school_product_id: 75506, status: "active")
 Edools::Enrollment.all
 ```
 
+### LessonProgress example
+
+Using `enrollment_id` as a parameter to search for LessonProgress:
+```ruby
+Edools::LessonProgress.find_paginated_from_enrollments(enrollment_id: 2075731)
```

Comment:
I would use

`find_paginated_by_enrollment`, in singular, you can't pass an array for example, and `by`, to be rails like.

Reply:
Agree.